src/comm.cc

```diff
@@ -471,6 +472,20 @@ comm_apply_flags(int new_socket,
         if ( addr.isNoAddr() )
             debugs(5,0,"CRITICAL: Squid is attempting to bind() port " << addr << "!!");
 
+#if defined(SO_REUSEPORT)
+        if (flags & COMM_REUSEPORT) {
+            int on = 1;
+            if (setsockopt(new_socket, SOL_SOCKET, SO_REUSEPORT, reinterpret_cast<char*>(&on), sizeof(on)) < 0) {
+                const auto savedErrno = errno;
+                const auto errorMessage = ToSBuf("cannot enable SO_REUSEPORT socket option when binding to ",
+                                                 addr, ": ", xstrerr(savedErrno));
+                if (reconfiguring)
```

Comment:
> Please use DBG_PARSE_NOTE() to escalate the debug message

 I am not sure we should do this because this comm_apply_flags() is not called in "-k parse" mode.

> Also please ensure the message is logged with "ERROR: " or "FATAL:

Since this situation is not fatal, I am not against adding "ERROR: " prefix to the message.

Reply:
I agree that `DBG_PARSE_NOTE` is inapropriate in post-configuration parsing code like this one.

Adding an `ERROR:` prefix to debugs() is a good idea. The exception catcher will (eventually) use the right prefix (which would depend on the catcher) so the throwing statement should not add any prefixes -- the thrower cannot (and should not try to) tell whether this is a fatal error.

Reply:
Done.